src/hotspot/share/cds/lambdaProxyClassDictionary.hpp

```diff
@@ -25,6 +25,7 @@
 #ifndef SHARED_CDS_LAMBDAPROXYCLASSINFO_HPP
 #define SHARED_CDS_LAMBDAPROXYCLASSINFO_HPP
 #include "cds/metaspaceShared.hpp"
+#include "classfile/javaClasses.hpp"
```

Comment:
Is this include necessary?

Reply:
It is needed, line 84:
 return java_lang_String::hash_code((const jbyte*)sym->bytes(), sym->utf8_length());

Reply:
I see.